indra/sources/phosphoelm/api.py

```diff
@@ -0,0 +1,107 @@
+import csv
+import logging
+import requests
+from unidecode import unidecode
+
+from .processor import PhosphoElmProcessor
+
+try:
+    from bs4 import BeautifulSoup
+    has_soup = False
+except ImportError:
+    BeautifulSoup = None
+    has_soup = False
+
+logger = logging.getLogger(__file__)
+
+s3_bucket = 'bigmech'
+ppelm_s3_key = 'phosphoELM_data/phosphoELM_all_2015-04.dump'
+kinases_list_web = 'http://phospho.elm.eu.org/kinases.html'
+
+
+def process_from_dump(fname=None, delimiter='\t'):
+    """Process a phospho.ELM file dump
+
+    fname : str
+        File path to the phospho.ELM file dump. If none is provided,
+        the file will be downloaded from S3.
+    delimiter : str
+        The delimiter to use for csv.reader
+
+    Returns
+    -------
+    ppp : indra.sources.phosphoELM.PhosphoElmProcessor
```

Comment:
The above line should just be `indra.sources.phosphoelm.PhosphoElmProcessor` (to fix that capitalization, also, the ppp : part is not needed, and it doesn't match the variable name).